RacingCar/main.swift

```diff
@@ -1,12 +1,27 @@
 //
-//  RacingCar - main.swift
-//  Created by yagom.
-//  Copyright © yagom. All rights reserved.
+//  main.swift
+//  RacingCar
+//
+//  Created by nylah.j on 2022/04/30.
 //
 
 import Foundation
 
-let input = InputView.input()
-let (operations, operands) = try InputParser.parse(input)
-let result = try Calculator().calculate(operators: operations, operands: operands)
-OutputView.show(output: result)
+let carCount = InputView.inputCarCount()
+if !InputChecker.isValid(carCount: carCount) {
```

Comment:
```suggestion
if InputChecker.isValid(carCount: carCount) == false {
```
저는 개인적으로 위처럼 명시하는 것을 선호합니다 😊